### ISSUE TITLE
Latex encoding escapes & in bibcode

### ISSUE THREAD

**Issue by marblestation** [adsabs/export_service#108]
If for instance you go to https://ui.adsabs.harvard.edu/abs/2017A%26A...604A.113B/exportcitation and specify the custom format:

```
%ZEncoding:latex\\bibitem{%R}\n%L: %T, %J, Vol. %V, pp. %p, %Y\n
```

You get back:

```
\bibitem{2017A\&A...604A.113B}
Bolmont, E., Gallet, F., Mathis, S., Charbonnel, C., Amard, L., and Alibert, Y.: Tidal dissipation in rotating low-mass stars and implications for the orbital evolution of close-in massive planets. II. Effect of stellar metallicity, Astronomy and Astrophysics, Vol. 604, pp. A113, 2017
```

The & should not be escaped in this context.

**Comment by golnazads** [comment]
fixed.

**Comment by golnazads** [comment]
released to dev on 6/4 and Build #260

```
\bibitem{2017A&A...604A.113B}
Bolmont, E., Gallet, F., Mathis, S., Charbonnel, C., Amard, L., and Alibert, Y.: Tidal dissipation in rotating low-mass stars and implications for the orbital evolution of close-in massive planets. II. Effect of stellar metallicity, Astronomy and Astrophysics, Vol. 604, pp. A113, 2017
```
